packages/docs/src/locales/en/en.json

```diff
@@ -1134,12 +1134,15 @@
     },
     "VaTimePicker": {
```

Comment:
Here are some examples of this:

Clicked on `8` hours, `9` hours are selected as a result.

![incorrecetTimeSelection](https://user-images.githubusercontent.com/7745899/170984832-53e64273-bdc4-4234-8c66-7bc818c88c2a.gif)